app/views/partials/personal_key/_key.html.erb

```diff
@@ -1,8 +1,8 @@
-<div class="bg-light-blue bg-personal-key">
```

Comment:
`bg-personal-key` and `bg-pk-box` styles already apply a background color, so I'm not sure why we would be using the utility classes as well. The conflict of CSS specificity is what was causing the visual brokenness on this page previously.

https://github.com/18F/identity-idp/blob/8361af01e95127c168998f5ed4c5fa91acc965c1/app/assets/stylesheets/components/_personal-key.scss#L45-L47

https://github.com/18F/identity-idp/blob/8361af01e95127c168998f5ed4c5fa91acc965c1/app/assets/stylesheets/components/_personal-key.scss#L52-L54